app/controllers/api/articles_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::ArticlesController < ApplicationController
+	def index
+		articles = Article.all_published
+		render json: articles, each_serializer: Articles::IndexSerializer
+	end
+end
```

Comment:
Massive indentation 😳

Reply:
This the indentation we use in the whole application. We use tabs, not spaces. Looks good in our VScode:
<img width="750" alt="screen shot 2019-01-22 at 20 07 42" src="https://user-images.githubusercontent.com/43814981/51559247-bbb00400-1e81-11e9-891a-9a6ef3d7614f.png">

Reply:
But not here. Please investigate.